lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -33,6 +33,21 @@
 
 TASK_LOG = logging.getLogger('edx.celery.task')
 
+ENROLLED_IN_COURSE = 'enrolled'
+
+NOT_ENROLLED_IN_COURSE = 'unenrolled'
+
+
+def _user_enrollment_status_mapping(course_id):
+    """
+    Returns a mapping of user ids to enrollment status in the given course.
+    """
+    return dict(CourseEnrollment.objects.values_list('user_id', 'is_active').filter(course_id=course_id))
```

Comment:
Also, you made me realize that `_enrollment_status()` really belongs in the `_CourseGradeReportContext` class.